packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -435,6 +437,26 @@ async function createPriceFeed(logger, web3, networker, getTime, config) {
       erc20Abi: ERC20.abi,
       blockFinder: getSharedBlockFinder(web3)
     });
+  } else if (config.type === "frm") {
+    const requiredFields = ["perpetualAddress"];
+    if (isMissingField(config, requiredFields, logger)) {
+      return null;
+    }
+
+    logger.debug({
+      at: "createPriceFeed",
+      message: "Creating FundingRateMultiplierPriceFeed",
+      config
+    });
+
+    return new FundingRateMultiplierPriceFeed({
+      ...config,
+      logger,
+      web3,
+      getTime,
+      perpetualAbi: Perpetual.abi,
+      blockFinder: getSharedBlockFinder(web3)
```

Comment:
Unrelated: How come we don't use `getSharedBlockFinder` in BalancerPriceFeed? Just haven't gotten around to it?

Reply:
It uses a different library for block caching. Specifically, it uses a library that pulls in _all_ blocks between two points for the purpose of computing a TWAP. The sharedBlockCache is using a library that does a binary search, and therefore produces a sparse block list.